troubleshooting/_posts/2016-12-12-my-build-is-failing-because-of-no-such-key-or-host-key-verification-failed-what-do-i-do.md

```diff
@@ -15,17 +16,20 @@ and the repository exists.
 ```
 
 To give containers within Runnable access to those dependency repositories, we just need to generate and add the right ssh key to the repository’s configuration.
- 
+
 ## Generate SSH Key {#generateSSH}
 
 SSH keys come in a pair, private and public. To generate a new ssh key pair, open up a terminal session on a Linux or Mac machine you trust. If you already have an SSH key that you would like to use, [skip this step](#addSSHGitHub).
 
 1. Navigate to the ~/.ssh/ directory. (Or create it, if it doesn't exist)
-  
-  `$ mkdir -p ~/.ssh/ && cd ~/.ssh`
+
+```
+$ mkdir -p ~/.ssh/ && cd ~/.ssh
+```
+
 
 2. Generate the SSH key
```

Comment:
This list is not setup properly, you gotta indent the code block before so markdown knows its all the same list.